sources/packages/web/src/components/students/applicationTracker/InProgress.vue

```diff
@@ -0,0 +1,296 @@
+    label="Parent income verification completed"
+    icon="fa:fas fa-check-circle"
+    icon-color="success"
+    content="The Canada Revenue Agency (CRA) has successfully verified your other parent's income."
+    v-if="applicationDetails?.parent2IncomeVerificationStatus?.success"
+  />
+
+  <application-status-tracker-banner
+    label="Partner income verification completed"
+    icon="fa:fas fa-check-circle"
+    icon-color="success"
+    content="The Canada Revenue Agency (CRA) has successfully verified your partner's income."
+    v-if="applicationDetails?.partnerIncomeVerificationStatus?.success"
+  />
+
+  <application-status-tracker-banner
+    label="StudentAid BC application review completed"
+    icon="fa:fas fa-check-circle"
+    icon-color="success"
+    content="StudentAid BC has successfully reviewed and accepted the additional information you provided."
+    v-if="
+      applicationDetails?.exceptionStatus ===
+      ApplicationExceptionStatus.Approved
+    "
+  />
+
+  <!-- Denial cards. -->
+  <application-status-tracker-banner
+    v-if="applicationDetails?.pirStatus === ProgramInfoStatus.declined"
+    label="Your institution denied your program information request"
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    background-color="error_bg"
+    ><template #content
+      ><span
+        ><strong>Reason from your institution:</strong>
+        {{ applicationDetails.PIRDeniedReason }}. Please contact the Financial
+        Aid Officer from your institution for more information. You will need to
+        edit your application for it to be processed again.</span
+      ></template
+    ></application-status-tracker-banner
+  >
+  <application-status-tracker-banner
+    v-if="
+      applicationDetails?.offeringStatus === OfferingStatus.ChangeOverwritten
+    "
+    label="Your institution updated the study period. Please re-submit your application with the updated study period."
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    background-color="error_bg"
+    ><template #content
+      ><span
+        >Your institution updated the study period information. Please
+        <a href="" @click="goToStudentApplication" class="primary-color"
+          >start a new application</a
+        >, select the updated study period, and submit your application to have
+        it processed again. Please contact the Financial Aid Officer from your
+        institution if you require more information about the change in study
+        period.</span
+      ></template
+    ></application-status-tracker-banner
+  >
+  <application-status-tracker-banner
+    v-if="
+      applicationDetails?.exceptionStatus ===
+      ApplicationExceptionStatus.Declined
+    "
+    label="StudentAid BC has not accepted your application"
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    content="StudentAid BC has not accepted 1 or more of the additional information that you provided in your 
```

Comment:
Such a long content can be returned though variable from script.

Reply:
mm, why. Vue is an HTML-based template, and it's pretty straight forward right? what is the befit of doing so?